### ISSUE TITLE
Passing the timeout as a positional argument is deprecated

### ISSUE THREAD

**Issue by espen** [mperham/sidekiq#5485]
```
Passing the timeout as a positional argument is deprecated, it should be passed as a keyword argument:
redis.brpop("queue:critical", "queue:batch", "queue:default", "queue:low", timeout: 2)(called from: /app/vendor/bundle/ruby/3.1.0/gems/sidekiq-6.5.5/lib/sidekiq/fetch.rb:49:in `block in retrieve_work')
```

There is also:
```
Redis#sadd will always return an Integer in Redis 5.0.0. Use Redis#sadd? instead.(called from: /app/vendor/bundle/ruby/3.1.0/gems/sidekiq-6.5.5/lib/sidekiq/launcher.rb:168:in `block (2 levels) in ❤')
```

Ruby version: 6.1.6.1
Rails version: 3.1.2
Sidekiq: 6.5.5
Deployed to: Heroku

**Comment by mperham** [comment]
Already fixed on main.

**Comment by kenbranchbob** [comment]
Hi @mperham, the same issue is still happening on my side.
```
2022-08-23T14:47:19.649892+00:00 app[worker.1]: Passing the timeout as a positional argument is deprecated, it should be passed as a keyword argument:
2022-08-23T14:47:19.649895+00:00 app[worker.1]:   redis.brpop("queue:images", "queue:mailers", "queue:imports", "queue:algolia", "queue:default", "queue:exports", timeout: 2)(called from: /app/vendor/bundle/ruby/2.7.0/gems/sidekiq-6.5.5/lib/sidekiq/fetch.rb:49:in `block in retrieve_work')
```

Ruby version: 2.7.4
Rails version: 6.1.6.1
Sidekiq: 6.5.5
Deployed to: Heroku

**Comment by spaquet** [comment]
Are you planning on a quick release 6.5.5.1 or 6.5.6 to fix this? Meanwhile I'm rolling back to 6.5.4

**Comment by mperham** [comment]
Not today. You need to roll back the Redis gem to 4.7.1, not Sidekiq.
